tools/github-issues/Azure.Sdk.Tools.GitHubIssues/Reports/CustomerReportedInvalidState/FindCustomerRelatedIssuesInvalidState.cs

```diff
@@ -50,7 +50,11 @@ private bool ValidateCustomerReportedIssues(ReportExecutionContext context, Repo
             tc.DefineTableColumn("Issues Found", i => i.Note);
 
             List<ReportIssue> issuesWithNotes = new List<ReportIssue>();
-            foreach (Issue issue in context.GitHubClient.SearchForGitHubIssues(CreateQuery(repositoryConfig)))
+
+            SearchIssuesRequest searchRequest = CreateQuery(repositoryConfig);
+            searchRequest.Type = IssueTypeQualifier.Issue;
```

Comment:
Late to the party 🙁.

Could you have changed this property in the CreateQuery method itself? It seems to only be used in this place in this file.

Reply:
Yeah ... good call, when I was parsing through the code I didn't realize that method was only used once. I'll tidy this up next time I am in there.